### PR TITLE
bumped holochain to 96815c179373bbc1ffca84c48e97c5a7746e4ead (Oct 27)

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

**Review by Connoropolous — APPROVED** [review]
🐈  purrfect. I was going to do this this morning, instead I wake up and it's done :)
it all looks right.

**Review by Connoropolous — CHANGES_REQUESTED** [review]
oops, ok wait. CI is failing. 
That's because of a test that checks the behavior of ZomeCallUnauthorized. Let me fix that one.

File: test/e2e/fixture/zomes/foo/Cargo.lock [hunk 1]
```diff
@@ -1,511 +0,0 @@
-# This file is automatically @generated by Cargo.
```
Comment (Connoropolous) [inline test/e2e/fixture/zomes/foo/Cargo.lock:1]: @zo-el why delete Cargo.lock?

Comment (Connoropolous) [reply test/e2e/fixture/zomes/foo/Cargo.lock:1]: Oh, I get it, because there's a top-level one now

Comment (zo-el) [reply test/e2e/fixture/zomes/foo/Cargo.lock:1]: That was an oversight.

**Review by Connoropolous — APPROVED** [review]
we've got CI passing now so re-approved

**Pull request merged**